docs/sub/visual_studio_2017.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+	<meta content="text/html; charset=ISO-8859-1" http-equiv="content-type">
+	<title>linux</title>
+</head>
+<body>
+	<h1>Building with Visual Studio 2017</h1>
+	Note:  It is assumed you install your third party libraries into C:\third_party.<br>

```

Comment:
Why do we have instructions of how to install the dependencies on Windows? If there are not special instructions to follow, we should not have those details. For instance, I am using Anaconda on Windows and I don't need to build the dependencies following these instructions. Incidentally, it is the same approach followed in the AppVeyor CI.